frontend/src/dashboard/dashboard.html

```diff
@@ -15,7 +15,11 @@ <h2 class="mt-4 mb-4 text-gray-900 font-semibold text-xl grow shrink">{{title}}<
         </button>
       </div>
     </div>
-    <div v-if="!showEditor" class="mt-4 mb-4">
+    <div  v-if="!showEditor" class="mt-4 mb-4">
+      <div>
+        <div>{{description}}</div>
+        <textarea readonly class="w-full p-2 mb-4 border border-gray-300 cursor-disable rounded-md focus:outline-none focus:ring-2 focus:ring-blue-500" rows="6" cols="6">{{code}}</textarea>
```

Comment:
Only suggestion is to remove this code editor. If we want to show the dashboard's code, it should go under the dashboard, and be in a Prisma `<pre>` rather than a Codemirror. But for now I don't think we should show the dashboard's code because primary consumers of dashboards are often non-technical users